### ISSUE TITLE
save=True in velocity_embedding_stream raises error

### ISSUE THREAD

**Issue by soerenab** [theislab/scvelo#891]
According to the [documentation](https://scvelo.readthedocs.io/scvelo.pl.velocity_embedding_stream/), `save` can be of type `bool`.
However, calling `scv.pl.velocity_embedding_stream(adata, basis='umap', save=True)` raises an error. 

Seems like a minor bug as one can workaround by passing a string.

```python
import scvelo as scv
scv.set_figure_params()
adata = scv.datasets.pancreas()
scv.pp.filter_and_normalize(adata, )
scv.pp.moments(adata, )
scv.tl.velocity(adata, mode='stochastic', )
scv.tl.velocity_graph(adata, n_jobs=7)
scv.pl.velocity_embedding(adata, basis='umap',)
scv.pl.velocity_embedding_grid(adata, basis='umap',)
scv.pl.velocity_embedding_stream(adata, basis='umap', save=True)
```

```pytb
TypeError                                 Traceback (most recent call last)
Input In [5], in <cell line: 1>()
----> 1 scv.pl.velocity_embedding_stream(adata, basis='umap', save=True)

File /usr/local/Caskroom/miniforge/base/envs/dyngene/lib/python3.8/site-packages/scvelo/plotting/velocity_embedding_stream.py:263, in velocity_embedding_stream(adata, basis, vkey, density, smooth, min_mass, cutoff_perc, arrow_color, arrow_size, arrow_style, max_length, integration_direction, linewidth, n_neighbors, recompute, color, use_raw, layer, color_map, colorbar, palette, size, alpha, perc, X, V, X_grid, V_grid, sort_order, groups, components, legend_loc, legend_fontsize, legend_fontweight, xlabel, ylabel, title, fontsize, figsize, dpi, frameon, show, save, ax, ncols, **kwargs)
    251 size = 8 * default_size(adata) if size is None else size
    252 ax = scatter(
    253     adata,
    254     layer=layer,
   (...)
    260     **scatter_kwargs,
    261 )
--> 263 savefig_or_show(dpi=dpi, save=save, show=show)
    264 if show is False:
    265     return ax

File /usr/local/Caskroom/miniforge/base/envs/dyngene/lib/python3.8/site-packages/scvelo/plotting/utils.py:997, in savefig_or_show(writekey, show, dpi, ext, save)
    995     ext = settings.file_format_figs
    996 filepath = f"{settings.figdir}{settings.plot_prefix}{writekey}"
--> 997 if "/" in writekey:
    998     filepath = f"{writekey}"
    999 try:

TypeError: argument of type 'NoneType' is not iterable
<summary> Error output </summary>
```
</details>

As far as I can see, the problem is that `writekey` is not set correctly if `save` is not a `str` in `savefig_or_show`.

<details> <summary> Versions </summary>

```pytb
scvelo==0.2.4  scanpy==1.9.1  anndata==0.8.0  loompy==3.0.6  numpy==1.21.6  scipy==1.8.1  matplotlib==3.5.2  sklearn==1.1.1  pandas==1.4.2  
```
</details>

**Comment by DriesSchaumont** [comment]
Hi! I stumbled upon the same issue and opened a PR

**Comment by WeilerP** [comment]
Fixed via #900.
